tests/group/group_async_work_group_copy_fp16.cpp

```diff
@@ -0,0 +1,43 @@
+/*******************************************************************************
+//
+//  SYCL 1.2.1 Conformance Test Suite
+//
+//  Copyright:	(c) 2017 by Codeplay Software LTD. All Rights Reserved.
+//
+*******************************************************************************/
+
+#include "group_async_work_group_copy_common.h"
+
+#define TEST_NAME group_async_work_group_copy_fp16
+
+namespace TEST_NAMESPACE {
+
+class TEST_NAME : public util::test_base {
+ public:
+  /** return information about this test
+  */
+  void get_info(test_base::info &out) const override {
+    set_test_info(out, TOSTRING(TEST_NAME), TEST_FILE);
+  }
+
+  /** execute the test
+  */
+  void run(util::logger &log) override {
+
+    auto queue = util::get_cts_object::queue();
+
+    if (!queue().get_device().has_extension("cl_khr_fp16")) {
+        log.note(
+            "Device does not support half precision floating point operations");
+        return;
+      }
```

Comment:
```suggestion
    if (!queue().get_device().has_extension("cl_khr_fp16")) {
      log.note(
            "Device does not support half precision floating point operations");
      return;
    }
```

Formatting in https://github.com/KhronosGroup/SYCL-CTS/pull/64/commits/5a18918e03698283b75cdb516bb77c301c76080c seems to be a bit off.